pkg/backup/item_collector.go

```diff
@@ -114,6 +116,16 @@ func (nt *nsTracker) init(
 	nt.logger = logger
 
 	for _, namespace := range unstructuredNSs {
+		ns := new(corev1api.Namespace)
+		if err := converter.FromUnstructured(namespace.UnstructuredContent(), ns); err != nil {
+			nt.logger.WithError(err).Errorf("Fail to convert unstructured into namespace %s", namespace.GetName())
+			continue
+		}
+		if ns.Status.Phase != corev1api.NamespaceActive {
+			nt.logger.Debugf("Skip namespace %s because it's not in Active phase.", namespace.GetName())
```

Comment:
Better be an info message b/c we are skipping namespaces which may be explicitly included in the backup CR

Reply:
Thanks. Modified.